src/backend/distributed/transaction/worker_transaction.c

```diff
@@ -117,6 +119,73 @@ SendCommandToWorkersWithMetadata(const char *command)
 }
 
 
+/*
+ * SendCommandToAllWorkers sends the given command to
+ * all workers.
+ */
+void
+SendCommandToAllWorkers(char *command)
+{
+	SendCommandListToAllWorkers(list_make1(command));
+}
+
+
+/*
+ * SendCommandListToAllWorkers sends the given command to all workers in
+ * a single transaction.
+ */
+void
+SendCommandListToAllWorkers(List *commandList)
+{
+	SendCommandListToAllWorkersInternal(commandList, true);
+}
+
+
+/*
+ * SendCommandListToAllWorkersInternal sends the given command to all workers in a single
+ * transaction. If failOnError is false, then it continues sending the commandList to other
+ * workers even if it fails in one of them.
+ */
+static void
+SendCommandListToAllWorkersInternal(List *commandList, bool failOnError)
+{
+	ListCell *workerNodeCell = NULL;
+	char *extensionOwner = CitusExtensionOwnerName();
```

Comment:
We've had some bugs from calling these functions with the expectation that they sent a command as the current user. It would be nice to either pass the user as a parameter, or add something like AsSuperuser to the name.